app/Resources/views/page.html.twig

```diff
@@ -189,6 +192,12 @@
 {% endfor %}
 {% endblock %}
 
+{% block stylesheets deferred %}
+{% for stylesheet in stylesheets %}
+    <link href="{{ asset(stylesheet) }}" rel="stylesheet">
```

Comment:
Ah yes, I was remembering the old `itemprop`-only spec (https://github.com/whatwg/html/commit/179983e9eb99efe417349a40ebb664bd11668ddd).